src/client/protocolgameparse.cpp

```diff
@@ -2319,7 +2319,11 @@ void ProtocolGame::parsePlayerStats(const InputMessagePtr& msg) const
 {
     const uint32_t health = g_game.getFeature(Otc::GameDoubleHealth) ? msg->getU32() : msg->getU16();
     const uint32_t maxHealth = g_game.getFeature(Otc::GameDoubleHealth) ? msg->getU32() : msg->getU16();
-    const uint32_t freeCapacity = g_game.getFeature(Otc::GameDoubleFreeCapacity) ? msg->getU32() / 100.f : msg->getU16() / 100.f;
+    uint32_t freeCapacity = g_game.getFeature(Otc::GameDoubleFreeCapacity) ? msg->getU32() : msg->getU16();
+    if (g_game.getClientVersion() > 760) {
```

Comment:
```suggestion
    if (g_game.getClientVersion() >= 772) {
```

<img width="329" height="249" alt="image" src="https://github.com/user-attachments/assets/c7124056-7ce9-4d05-b985-777e54959639" />